Agrego soporte de php 7.1 y soluciono problema entrypoint

chrodriguez opened the pull request:
Para poder trabajar en arai usuarios necesito la imagen de php 7.1 Por eso avancé con el refactor que disparó:

* El hallazgo de un error con los permisos en el entrypoint
* Hacer funcionar el workflow con más versiones de php
* Actualizar el readme

chrodriguez committed:
Arreglo permisos en entrypoint

Todavía no entiendo como subí mi commit anterior y no me di cuenta de
este problema!

Ya quedó con permisos de ejecución en el entrypoint

chrodriguez committed:
Refactor de imagen 7.1

La cambié bastante, pero manteniendo la línea de la 7.4, aunque tuve que
preservar la lógica de agergar el módulo de sodium

chrodriguez committed:
Actualizo actions para contemplar imagen php 7.1

Ahora se considera además de la 7.4 la 7.1

chrodriguez committed:
Marco en README estado actual

Marco que ya soportamos php 7.1 y 7.4

enfoqueNativo commented on 7.4-alpine/Dockerfile:
```diff
@@ -17,6 +17,7 @@ RUN apk --no-cache add \
     php7-iconv \
     php7-json \
     php7-mbstring \
+    php7-mcrypt \
```
Hay algun motivo puntual para tener esto aca?, si es solo c&p la eliminaria.. es una extension descontinuada.

chrodriguez replied:
Porque estaba en la 7.1 e hice merge. Lo saco de ambos?

sergiovier replied:
Es un requerimiento de las imágenes PHP 7.1 por las librerías que existían ahí. A partir de PHP 7.2 ya NO se usa. 

Entonces, debería estar solo en PHP 7.1, manteniendo ese bc con las apps existentes...

chrodriguez replied:
Listo, quité el módulo en la versión 7.4

chrodriguez committed:
Quito mcrypt en 7.4

Lo había agregado porque estaba en la versión 7.1. Sin embargo, al
realizar un PR previo, @sergiovier me indicó que en 7.4 no es necesario

chrodriguez committed:
Fix 7.1 apache2 requiere directorio run

En 7.4 se crea por el instalador, pero no en alpine usado para 7.1

chrodriguez commented:
Arreglé otro problema de último momento que detecté con arai-docs
